### ISSUE TITLE
Feature Request - syntax highlighting for cite keys outside square brackets

### ISSUE THREAD

**Issue by doctornerdis** [renerocksai/sublime_zk#46]
Right now, sublime_zk changes the syntax coloring for cite keys that appear in square brackets: e.g. `[@smith_book_2018]`. But would it be possible to have the same coloring for cite keys that are not inside brackets?

For example, the two citation examples below:

```
Lorem @smith_book_2018 ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et @smith_article_2012 [p. 32] dolore magna aliqua.
```

**Comment by renerocksai** [comment]
Dr. Nerdis,

That is certainly doable. In fact, it's done 😄 

It only works with pandoc notation, as multimarkdown's `#citekey` notation would make it indistinguishable from `#tags`. But as we're both pandoc users, that should be fine 😄

**Comment by doctornerdis** [comment]
Many thanks. This looks great! I actually spent the weekend writing a python script to reformat all my notes so that I can use all of sublime_zk's features. I'm so glad I did!

**Comment by renerocksai** [comment]
Wow, that is awesome! I am so glad I got introduced to python; has made my
life so much easier!
Cool your notes are compatible now! But let me know if sublime_zk needs to
become more flexible... 😀

Am 20.02.2018 19:38 schrieb "doctornerdis" <notifications@github.com>:

Many thanks. This looks great! I actually spent the weekend writing a
python script to reformat all my notes so that I can use all of
sublime_zk's features. I'm so glad I did!

—
You are receiving this because you modified the open/close state.

Reply to this email directly, view it on GitHub
<https://github.com/renerocksai/sublime_zk/issues/46#issuecomment-367075354>,
or mute the thread
